### PR TITLE
Update to latest ouroboros-network

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

**Review by erikd — APPROVED** [review]

File: cardano-node/src/Cardano/Tracing/Tracers.hs [hunk 1]
```diff
@@ -502,6 +502,10 @@ mkConsensusTracers trSel verb tr nodeKern fStats = do
               traceI trmet meta "submissions.rejected.count" =<<
                 STM.modifyReadTVarIO tSubmissionsRejected (+ ptxcRejected processed)
 
+            TraceLabelPeer _ TraceTxInboundTerminated -> return ()
+            TraceLabelPeer _ (TraceTxInboundCanRequestMoreTxs _) -> return ()
+            TraceLabelPeer _ (TraceTxInboundCannotRequestMoreTxs _) -> return ()
```
Comment (newhoggy) [inline cardano-node/src/Cardano/Tracing/Tracers.hs:507]: @coot Mainly I wanted to know if something should be done here.

**Comment by newhoggy** [comment]
bors merge

File: cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs [hunk 2]
```diff
@@ -156,6 +156,7 @@ instance HasSeverityAnnotation (TraceChainSyncClientEvent blk) where
   getSeverityAnnotation (TraceFoundIntersection _ _ _) = Info
   getSeverityAnnotation (TraceRolledBack _) = Notice
   getSeverityAnnotation (TraceException _) = Warning
+  getSeverityAnnotation (TraceTermination _) = Warning
```
Comment (coot) [inline cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs:159]: I don't think other mini-protocols use `Warning` for tracing termination.  Terminating min-protocols will be a normal mode of operation, so `Warning` seems to high severity for me.  Could you check what other mini-protocols are using?

File: cardano-node/src/Cardano/Tracing/Tracers.hs (hunk 1 above)
Comment (coot) [reply cardano-node/src/Cardano/Tracing/Tracers.hs:507]: No, I don't think we need to do anything here.

File: cardano-node/src/Cardano/Tracing/Tracers.hs [hunk 3]
```diff
@@ -502,6 +502,10 @@ mkConsensusTracers trSel verb tr nodeKern fStats = do
               traceI trmet meta "submissions.rejected.count" =<<
                 STM.modifyReadTVarIO tSubmissionsRejected (+ ptxcRejected processed)
```
Comment (coot) [inline cardano-node/src/Cardano/Tracing/Tracers.hs:503]: I know it's unrelated to the patch here, but using `STM` for counters will have performance penalty. Using something like `ekg`'s `Gauge` for that should be better.

**Pull request merged**